tests/longevity-2mv-backpressure-4d.yaml

```diff
@@ -0,0 +1,77 @@
+test_duration: 5760
```

Comment:
Make it a bit bigger than 5760 which is the duration of the stress.
It should be test duration = setup time + prepare write time + stress time + short buffer.

Reply:
Changed to 4.5 days (6480 min)